package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "Base",
-  "version": "2.1.0",
+  "version": "3.0.0",
```

Comment:
Since the actual core of base (the SASS structure) hasn't changed I think this is a minor release and 2.2.0 would be ok/